src/App.scss

```diff
@@ -1,2 +1,102 @@
 @use "theme/utilities.scss";
 @use "theme/global.scss";
+// @use "theme/fonts.scss";
+
+
+.App{
+    background-color: #8ecae6;
+}
+
+.c-site-header{
+  padding: 20px;
+}
+
+.header{
+  background-color: #219ebc;
+  padding: 10px;
+}
+
+.Location{
+  background-color: #37b0cf;
+  margin: 10px;
+  padding: 10px;
+  border: none;
+  border-bottom: 2px solid rgb(8, 8, 78);
+}
+
+.button{
+  background-color: #0d6b9a;
+  color: white;
+  margin-left: 15px;
+  padding: 15px;
+  border: none;
+  border-radius: 5px;
+}
+
+// ----------TodayForecast section----------
+.weather-img{
+  margin: auto;
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+}
+
+.description{
```

Comment:
One way to avoid duplicating these two lines is to declare a utility class eg -
```
.center-horizontal {
  display: flex;
  justify-content: center;
}
```

Then use that class on the 3 elements in addition to .description, .temp and .current-container